src/main/java/bio/terra/cda/app/service/QueryService.java

```diff
@@ -20,10 +20,12 @@
 
   public static class QueryResult {
 
+    public final Query query;
```

Comment:
This appears to be unnecessary as it's not used by the caller. As it is, the code here doesn't process it, it only returns it unmodified to the caller, so there doesn't seem to be a need to pass it through.